src/config/fabric8-ui.env.js

```diff
@@ -3,9 +3,9 @@ window.Fabric8UIEnv = {
   "analyticsWriteKey": "{{ .Env.ANALYTICS_WRITE_KEY }}",
   "forgeApiUrl": "{{ .Env.FABRIC8_FORGE_API_URL }}",
   "openshiftConsoleUrl": "{{ .Env.OPENSHIFT_CONSOLE_URL }}",
+  "openshiftProxiedApiServer": "{{ .Env.PROXIED_K8S_API_SERVER }}",
   "witApiUrl": "{{ .Env.FABRIC8_WIT_API_URL }}",
   "ssoApiUrl": "{{ .Env.FABRIC8_SSO_API_URL }}",
-  "forgeApiUrl": "{{ .Env.FABRIC8_FORGE_API_URL }}",
```

Comment:
the line I removed was a duplicate (that you'd spotted in an early PR :) both keys were `forgeApiUrl` so I think we can safely delete this one?